src/index.ts

```diff
@@ -73,86 +73,100 @@ declare global {
-       * A list of file extensions (in priority order) that Graphile Worker
-       * should attempt to import directly when loading tasks. Defaults to
-       * `[".js", ".cjs", ".mjs"]`.
+       * A list of file extensions (in priority order) that Graphile Worker should
+       * attempt to import directly when loading task executors from the file system.
+       *
+       * @defaultValue `[".js", ".cjs", ".mjs"]`
        */
       fileExtensions?: string[];
-
       /**
        * How long in milliseconds after a gracefulShutdown is triggered should
-       * we wait to trigger the AbortController, which should cancel supported
-       * asynchronous actions?
+       * Graphile Worker wait to trigger the AbortController, which should cancel
+       * supported asynchronous actions?
        *
-       * @defaultValue `5000`
+       * @defaultValue `5_000`
        */
       gracefulShutdownAbortTimeout?: number;
-
       /**
-       * Set `true` to use the time as recorded by Node.js rather than
+       * Set to `true` to use the time as recorded by Node.js rather than
        * PostgreSQL. It's strongly recommended that you ensure the Node.js and
        * PostgreSQL times are synchronized, making this setting moot.
+       *
+       * @defaultValue `false`
        */
       useNodeTime?: boolean;
-
       /**
        * **Experimental**
        *
-       * How often should we scan for jobs that have been locked too long and
-       * release them? This is the minimum interval, we'll choose a time between
-       * this and `maxResetLockedInterval`.
+       * How often should Graphile Worker scan for and release jobs that have been
+       * locked too long? This is the minimum interval in milliseconds. Graphile
+       * Worker will choose a time between this and `maxResetLockedInterval`.
+       *
+       * @defaultValue `480_000`
        */
       minResetLockedInterval?: number;
       /**
        * **Experimental**
        *
-       * The upper bound of how long we'll wait between scans for jobs that have
-       * been locked too long. See `minResetLockedInterval`.
+       * In milliseconds, the upper bound of how long Graphile Worker will wait
+       * between scans for jobs that have been locked too long (see
+       * `minResetLockedInterval`).
+       *
+       * @defaultValue `600_000`
        */
       maxResetLockedInterval?: number;
-
       /**
        * **Experimental**
        *
-       * When getting a queue name in a job, we batch calls for efficiency. By
-       * default we do this over a 50ms window; increase this for greater efficiency,
-       * reduce this to reduce the latency for getting an individual queue name.
+       * The window size in milliseconds in which Graphile Worker batches calls for
+       * getting a queue name in a job. This batching is done for efficiency. Increase
+       * this window for greater efficiency. Reduce this window to reduce the
+       * latency for getting an individual queue name.
```

Comment:
```suggestion
       * The size, in milliseconds, of the time window over which Graphile Worker
       * will batch requests to retrieve the queue name of a job. Increase the size
       * of this window for greater efficiency, or reduce it to improve latency.
```

Reply:
Thanks for updating this one. Still not exactly sure what this one does. Got it on my todo list to dive in and figure out :)